src/main/java/racing/view/OutputView.java

```diff
@@ -1,30 +1,51 @@
 package racing.view;
 
-import racing.dto.RacingResultDto;
+import racing.domain.Car;
+import racing.dto.RacingGameResult;
 
 import java.util.List;
 
 public class OutputView {
+    private static final String HYPHEN = "-";
+    private static final String COLON = " : ";
 
-    public OutputView() {
-        this.printGuidance();
-    }
-
-    private void printGuidance() {
+    public void printGuidance() {
         System.out.println("실행 결과");
     }
 
-    public void printRacingResult(RacingResultDto racingResultDto) {
-        List<Integer> racingPositions = racingResultDto.calculateRacingResult();
-        racingPositions.forEach(position -> this.outputCarPosition(position));
+    public void printRacingResult(RacingGameResult racingGameResult) {
+        List<Car> cars = racingGameResult.getCars();
+        cars.forEach(car -> this.outputCarPosition(car.getName(), car.findCurrentPosition()));
         System.out.println();
     }
 
-    private void outputCarPosition(Integer racingPosition) {
+    private void outputCarPosition(String name, int position) {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append(name);
+        stringBuilder.append(COLON);
+        for (int i = 0; i < position; i++) {
+            stringBuilder.append(HYPHEN);
+        }
+        System.out.println(stringBuilder);
+    }
+
+    public void printRacingWinners(RacingGameResult racingGameResult) {
+        List<Car> winners = racingGameResult.calculateWinners();
         StringBuilder stringBuilder = new StringBuilder();
-        for (int i = 0; i < racingPosition; i++) {
-            stringBuilder.append("-");
+        for (Car car : winners) {
```

Comment:
```suggestion
    winners.stream()
    .map(car -> car.getName())
    .collect(Collectors.joining(", "));
```
이렇게 스트림을 이용해주게 되면 `removeLastColon()` 을 하지 않을 수 있어요.